### ISSUE TITLE
EXPERIMENTO - Estudiar cómo y pasar una red entrenada en PyTorch a TensorFlow y TensorFlow Lite

### ISSUE THREAD

**Issue by giorg0s** [giorg0s/2020-tfg-jorge_guerrero#6]

**Comment by giorg0s** [comment]
Según comenta Andrés los pasos a seguir podrían ser:

1. Convertir Resnet pre-entrenada.
2. Convertir MovilenetV2 pre-entrenada.
3. Convertir una deconvolucional sin entrenar.

**La conversión idónea debería ser de Pytorch 1.6 a TF 1.14-15 o TF 2.**

**Comment by giorg0s** [comment]
https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html

**Comment by giorg0s** [comment]
https://medium.com/@santhoshkolloju/convert-your-pytorch-models-to-tensorflow-with-onnx-84c3bdd8d722

**Comment by giorg0s** [comment]
Al acabar el proceso PyTorch -> ONNX -> TF y visualizar el grafo resultante con tensorboard lo que obtengo es esto (concretamente usando un modelo de mobilenet_v2 pre-entrenado):
![image](https://user-images.githubusercontent.com/5454012/98148739-4d5cfa00-1ecd-11eb-8208-c5cc4f6277f3.png)
Creo que no tiene mucha pinta de haberlo hecho bien.

**Comment by jmbuena** [comment]
Parece algo raro, sí. Lo suyo sería mirar la arquitectura de red de mobilenet_v2 ya implementada en tensorflow y ver como es.

**Comment by giorg0s** [comment]
> Parece algo raro, sí. Lo suyo sería mirar la arquitectura de red de mobilenet_v2 ya implementada en tensorflow y ver como es.

Sí, creo que el problema está cuando paso el modelo de ONNX a tensorflow, porque al visualizarlo tiene sentido:

![mobilenet_v2_onnx](https://user-images.githubusercontent.com/5454012/98155985-6d91b680-1ed7-11eb-99f2-98f9c2864628.png)
